tests/users/test_data.py

```diff
@@ -174,3 +174,18 @@ async def test_find_or_create_b2c_user(
         assert int(user.handle.split("-")[-1])
 
     assert user == snapshot(matcher=path_type({"handle": (str,)}))
+
+
+class TestUserExistenceChecks:
```

Comment:
```suggestion
class TestCheckUsersExist:
```
Class name should closely match function name being tested